src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -51,11 +51,13 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
   const { numFacilities } = props;
   const updatedAtDate = Number(scenario?.updatedAt);
 
-  const handleScenarioChange = (scenarioChange: object) => {
-    const changes = Object.assign({}, scenario, scenarioChange);
-    saveScenario(changes).then((_) => {
-      dispatchScenarioUpdate(changes);
-    });
+  const handleScenarioChange = (scenarioChange: any) => {
+    if (scenarioChange.name || scenarioChange.description) {
+      const changes = Object.assign({}, scenario, scenarioChange);
+      saveScenario(changes).then((_) => {
```

Comment:
@sokunthel I believe this is why the build is marked as currently failing. This tripped up both @sprad and myself on previous PRs. We had tried using an inline comment to disable the lint check but to no avail. Would be nice if you could take a quick look and see if you can figure out how to make lint not fail on this, but I wouldn't spend much time on it!

Reply:
Found the solution. Added to eslint json. Let me know should you prefer inline comment.

Reply:
Perfect. Thanks, @sokunthel